synapse/models/inet.py

```diff
@@ -1390,63 +1390,77 @@ def getModelDefs(self):
                     }),
 
                     ('inet:web:acct', ('comp', {'fields': (('site', 'inet:fqdn'), ('user', 'inet:user')), 'sepr': '/'}), {
+                        'deprecated': True,
                         'doc': 'An account with a given Internet-based site or service.',
```

Comment:
We generally try to provide a doc update to something like `Deprecated. Please use inet:service:account.` or `Deprecated. Please use the inet:service:* forms` ( if there's not a one-to-one replacement )